Bottom: 33ded6f410a0975429354454e973a1eec6968f0e
Top:    56409c2ad660f0717dc54fefed8e388c01fa8a01
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-13 18:15:45 +0000

Refresh of add-polygon-parentage-tree

---

diff --git a/src/polygon1.c b/src/polygon1.c
index acb59e6..5daaf23 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -158,6 +158,9 @@ pline_dump (VNODE * v)
     }
   while ((v = NEXT_VERTEX(v)) != s);
 }
+#endif
+
+static void pline_dump (VNODE * v);
 
 /*static */void
 poly_dump (POLYAREA * p)
@@ -178,7 +181,6 @@ poly_dump (POLYAREA * p)
     }
   while ((p = p->f) != f);
 }
-#endif
 
 static VNODE *
 poly_CreateNodeFull (Vector v, bool is_round, Coord cx, Coord cy, Coord radius)
